ospfd/ospf_routemap.c

```diff
@@ -462,6 +464,115 @@ static const struct route_map_rule_cmd route_set_metric_cmd = {
 	route_set_metric_free,
 };
 
+/* `set min-metric METRIC' */
+/* Set min-metric to attribute. */
+static enum route_map_cmd_result_t
+route_set_min_metric(void *rule, const struct prefix *prefix, void *object)
+{
+	uint32_t *min_metric;
+	struct external_info *ei;
+
+	/* Fetch routemap's rule information. */
+	min_metric = rule;
+	ei = object;
+
+	ei->min_metric = *min_metric;
+
+	if (ei->min_metric > OSPF_LS_INFINITY)
+		ei->min_metric = OSPF_LS_INFINITY;
+
+	if ((uint32_t)ROUTEMAP_METRIC(ei) < ei->min_metric)
+		ROUTEMAP_METRIC(ei) = ei->min_metric;
+
+	return RMAP_OKAY;
+}
+
+/* set min-metric compilation. */
+static void *route_set_min_metric_compile(const char *arg)
+{
+
+	uint32_t *min_metric;
```

Comment:
Why not just `uint32_t min_metric`?

Reply:
just being consistent with other functions in vicinity! :)